### PR TITLE
Add Chia.app v1.0.5

### PR DESCRIPTION
**Pull request by starcatmeow**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).
- [x] `brew audit --cask {{cask_file}}` is error-free.
- [x] `brew style --fix {{cask_file}}` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew install --cask {{cask_file}}` worked successfully.
- [x] `brew uninstall --cask {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/chia.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+cask "chia" do
+  version "1.0.5"
+  sha256 "d77ae2e3e7187d892a8da1e9be1700d204da46c0e1151f44f3d0ec61adff7524"
+
+  url "https://github.com/Chia-Network/chia-blockchain/releases/download/#{version}/Chia-#{version}.dmg",
+      verified: "github.com/Chia-Network/chia-blockchain/"
+  name "Chia Blockchain"
+  desc "GUI Python Implementation for"
```
Comment (singularitti) [inline Casks/chia.rb:8]: "GUI Python Implementation for" what?

Comment (starcatmeow) [reply Casks/chia.rb]: Sorry for that, now I fixed.

**Comment by miccal** [comment]
Thank you for your contribution to `homebrew-cask` @starcatmeow.

Could you please try to add a [`zap` stanza](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/zap.md), thanks.

**Comment by starcatmeow** [comment]
Thank you. Added.

**Comment by miccal** [comment]
Thank you @starcatmeow.

**Review by miccal — APPROVED** [review]

**Pull request merged**